big-o-notation/big-o.js

```diff
@@ -0,0 +1,44 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {};                  // 1 * 1 = O(1)
+  const unique = [];                // 1 * 1 = O(1)
+  for (
+    let i = 0;                      // 1 * 1 = O(1)
+    i < words.length;               // 2 * n = O(2n) ~= O(n)
+    i++                             // 2 * n = O(2n) ~= O(n)
+  ) {
+    const word = words[i];          // 2 * n = O(2n) ~= O(n)
+    if (!seen[word]) {              // 3 * n = O(3n) ~= O(n)
+      seen[word] = true;            // 2 * n = O(2n) ~= O(n)
+      unique[unique.length] = word; // 3 * n = O(3n) ~= O(n)
+    }
+  }
+  return unique;                    // 1 * 1 = O(1)
+} // Big O Notation for uniqueLinear: O(n)
+
+function uniqueQuadratic(words) {
+  const unique = [];                // 1 * 1 = O(1)
+  for (
+    let i = 0;                      // 1 * 1 = O(1)
+    i < words.length;               // 2 * n = O(2n) ~= O(n)
+    i++                             // 2 * n = O(2n) ~= O(n)
+  ) {
+    const word = words[i];          // 2 * n = O(2n) ~= O(n)
+    let isUnique = true;            // 1 * 1 = O(1)
```

Comment:
This statement is in the context of a loop block, so it will be O(n) in this case